core/types/backwards_compat.libevm_test.go

```diff
@@ -151,6 +231,14 @@ func (e *cChainBodyExtras) RLPFieldPointersForDecoding(b *Body) *rlp.Fields {
 	}
 }
 
+func (e *cChainBodyExtras) BlockRLPFieldsForEncoding(b *BlockRLPProxy) *rlp.Fields {
+	panic("unimplemented")
```

Comment:
nit for the sake of having a full example to use in our existing consumers (also I like copy pasting)

```suggestion
	return &rlp.Fields{
		Required: []any{b.Header, b.Txs, b.Uncles, b.Version, rlp.Nillable(b.ExtData)},
	}
```

Reply:
As in https://github.com/ava-labs/libevm/pull/133/files#r1954477090

This isn't an example, it's a test with a specific intent.